src/pages/UserHome.jsx

```diff
@@ -35,6 +38,18 @@ const useStyles = makeStyles((theme) => ({
   addScholarship: {
     marginTop: theme.spacing(6),
   },
+  addTextAndLink: {
+    textAlign: 'center',
+  },
+  addIconSpacing: {
+    marginTop: theme.spacing(6),
+  },
+  centerTextAndLink: {
```

Comment:
This is descriptive of the specific child elements of the element styled but maybe a better name might be `noScholarshipsBox` as it's the `Box` containing elements for when there's no scholarships.